docs/api.rst

```diff
@@ -16,6 +16,13 @@ Adaptador Base
 Adaptadores
 -----------
 
+CpiAllUrbanCityAverage (FED's Consumer Price Index for All Urban Consumers: All Items (CpiAllUrbanCityAverage))
```

Comment:
Why is _CpiAllUrbanCityAverage_  repeated in the beginning and ending of this line?